core/chain-configs/src/client_config.rs

```diff
@@ -29,11 +36,19 @@ pub struct GCConfig {
     /// when cleaning forks during garbage collection.
     #[serde(default = "default_gc_fork_clean_step")]
     pub gc_fork_clean_step: u64,
+
+    /// Number of epochs for which we keep store data.
+    #[serde(default = "default_num_epochs_to_keep_store_data")]
+    pub num_epochs_to_keep_store_data: u64,
```

Comment:
I really think we should call this `gc_num_epochs_to_keep` to be honest. The current name is long and confusing. And the `gc_` prefix would be to fit with the other fields. We at least could use `serde(rename = "...")` to have a name with `gc_` prefix.

Reply:
Nice call. Changed all the names